components/training/eventBooking.tsx

```diff
@@ -167,6 +168,24 @@ const addRightBorder = (index) => {
   return (index + 1) % 3 != 0 ? "md:border-r-8" : "";
 };
 
+const EventDates = ({ duration, date }) => {
+  const startDate = dayjs(date);
+  const endDate = startDate.add(duration, "day");
+
+  const eventDateRange = () => (
+    <>
+      {startDate.format("Do")} - {endDate.format("Do MMMM YYYY")}
+      <div>
+        ({startDate.format("ddd")} - {endDate.format("ddd")})
+      </div>
+    </>
+  );
+
+  const eventDate = () => startDate.format("Do (ddd) MMMM YYYY");
```

Comment:
i'd recommend adding a comment to show what the expected output should be from this function

Reply:
Done